forecastiopy/ForecastIO.py

```diff
@@ -6,12 +6,16 @@
 The resulting object is used by the other classes to get the information.
 """
 
-from __future__ import print_function
 import sys
 import json
 import requests
+import logging
 
 
+log = logging.getLogger(__name__)
+log.setLevel(level=logging.WARNING)
+log.addHandler(logging.NullHandler())
```

Comment:
According to the [Hitchhiker's Guide to Python](http://docs.python-guide.org/en/latest/writing/logging/#logging-in-a-library) I would move the `NullHandler` into the `__init__.py` file:

```python
import logging
logging.getLogger(__name__).addHandler(logging.NullHandler())
```